components/sysutils/nut/history

```diff
@@ -0,0 +1,3 @@
+system/management/nut-common@2.7.4,5.11-2022.0.0.6 system/management/nut/common
+system/management/nut/bins@2.7.4,5.11-2022.0.0.6 system/management/nut/server system/management/nut/clients
+
```

Comment:
Please remove the empty line here.

Reply:
Yeah, and also a second package name, that did not work.

Reply:
I guess it is better to get a new `nut-bins` that just depends on server and client.

Reply:
> I guess it is better to get a new `nut-bins` that just depends on server and client.

Like a meta-package?  Yes, looks so.

Reply:
> Yeah, and also a second package name, that did not work.

Yes, obviously.  FYI, the `history` file is processed by the [generate-history](https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/tools/generate-history) awk script.